### PR TITLE
Post put

### LINKED ISSUE #14: POST /api/ad (BE)
- [x] Move jwtClient to be shared between all routes
- [x] grab created/modified dates

### PR DESCRIPTION
**Pull request by chaynsaw**

### CODE REVIEW

File: server/yarn.lock [hunk 1]
```diff
@@ -1106,6 +1106,21 @@ minimist@^1.2.0:
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
   integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
 
+<<<<<<< HEAD
```
Comment (raymond-sde) [inline server/yarn.lock]: Looks unintentional

File: server/yarn.lock [hunk 2]
```diff
@@ -1106,6 +1106,21 @@ minimist@^1.2.0:
...
+  dependencies:
+    minimist "^1.2.5"
+
+moment@^2.29.1:
+  version "2.29.1"
+  resolved "https://registry.yarnpkg.com/moment/-/moment-2.29.1.tgz#b2be769fa31940be9eeea6469c075e35006fa3d3"
+  integrity sha512-kHmoybcPV8Sqy59DwNDY3Jefr64lK/by/da0ViFcuA4DH0vQg5Q6Ze5VimxkfQNSC+Mls/Kx53s7TjP1RhFEDQ==
+
+=======
+>>>>>>> cd01a4bfdc7eab91104e3c0a869fe4cbe05e53f5
```
Comment (raymond-sde) [inline server/yarn.lock]: Looks unintentional

File: server/routes/ads.js [hunk 3]
```diff
@@ -1,25 +1,12 @@
...
-  }
-});
-
+const jwtClient = require('../bin/jwtClient.js');
 const spreadsheetId = process.env.SPREADSHEET_ID;
 const sheets = google.sheets('v4');
-let sheetRange = 'Ads!A2:J10';
+const uuidv4 = require('uuid').v4;
+const moment = require('moment');
+let sheetRange = 'Ads!A2:J';
```
Comment (raymond-sde) [inline server/routes/ads.js:9]: will this always insert in the last row?

Comment (chaynsaw) [reply server/routes/ads.js:9]: Yeah, far as I know. Screenshot from the docs
image

File: server/yarn.lock (hunk 1 above)
Comment (chaynsaw) [reply server/yarn.lock]: God damn it.

File: server/package.json [hunk 4]
```diff
@@ -15,7 +15,8 @@
     "googleapis": "~71.0.0",
     "http-errors": "~1.6.3",
     "morgan": "~1.9.1",
-    "request": "^2.88.2"
+    "request": "^2.88.2",
+    "uuid": "^8.3.2"
```
Comment (raymond-sde) [inline server/package.json:20]: i saw moment library in your yarn.lock file. not sure why its not in your package.json file. you might have library conflicts. i recommend reinstalling your new packages and committing again

Comment (chaynsaw) [reply server/package.json:20]: Added it back. Nice catch.

**Pull request merged**